docs/development.md

```diff
@@ -92,6 +92,19 @@ Within this range, each package in the API package is given a range of 100 locat
 
 ## API Notes
 
+### API File Structure
+
+All API endpoints must reside in a directory that follows the same structure as the endpoint's base path from `internal/api`.
+For example, the endpoint `users/list` must reside in a directory called `internal/api/users`.
+Additionally, the handler for the endpoint must be in a file with the same name.
+For example, the handler for `users/list` must be in a file named `list.go`.
+Finally, the handler function must contain the substring `Handle`.
+For our `users/list` example, a good handler name would be `HandleList`.
```

Comment:
This paragraph is a bit "stuttery",
in that it gives an example and then lists another requirement,
and then repeats this.

Instead, try having the description first then then the example.
For example:
> The structure of API endpoints files must follow these rules ...
> \1) ...
> ...
> For example, the handler for `users/list` is called ... and located at ...

(Ignore the backslash above.)